src/Widgets/SliderWidget/SliderWidgetEditingConfig.ts

```diff
@@ -21,7 +21,8 @@ provideEditingConfig(SliderWidget, {
     },
     minHeight: {
       title: 'Minimum height (in px)',
-      description: 'Default: 400',
+      description:
+        'Maintains consistent slide appearance when content varies in height. Default: 400',
```

Comment:
* [ ] FYI When -> if? E.g., if all slides contain just an image and all images have the same height, this can be ignored. On the other hand, maybe this is an unrealistic scenario, so "when" is maybe more to the point.